src/libcrun/seccomp.c

```diff
@@ -247,13 +247,17 @@ int
 libcrun_generate_seccomp (libcrun_container_t *container, int outfd, unsigned int options, libcrun_error_t *err)
 {
 #ifdef HAVE_SECCOMP
-  runtime_spec_schema_config_linux_seccomp *seccomp = container->container_def->linux->seccomp;
+  runtime_spec_schema_config_linux_seccomp *seccomp;
```

Comment:
Should this init to NULL?

Reply:
it is initialized few lines below (set to `container->container_def->linux->seccomp`)

Reply:
Sure but you open the risk to future you using it before it is initialized.